fxa-client/src/http_client/mod.rs

```diff
@@ -244,6 +244,21 @@ impl<'a> Client<'a> {
         Client::make_request(request)?.json().map_err(|e| e.into())
     }
 
+    pub fn destroy_oauth_token(&self, token: &str) -> Result<()> {
+        let body = json!({
+            "token": token,
+        });
+        let url = self.config.oauth_url_path("v1/destroy")?;
+        let client = ReqwestClient::new();
+        let request = client
+            .request(Method::POST, url)
+            .header(header::CONTENT_TYPE, "application/json")
+            .body(body.to_string())
+            .build()?;
+        Client::make_request(request)?;
+        Ok(())
```

Comment:
We don't want to propagate destroy errors or can we log that it failed but still `Ok` ?

Reply:
`?` will still propagate errors, however we do ignore that error up in the call chain (we might still want to log this in).